### PR TITLE
change rio mount point to /retirement-income-opt..

### PR DESCRIPTION
**Pull request by asmega**
/retirement-income-options

### CODE REVIEW

File: spec/controllers/mount_controller_spec.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   describe '#alternate_url' do
     it 'returns an alternate_url if one is available' do
-      expect(subject).to receive(:url_for).with('script_name' => '/cy/alternative_engine')
+      expect(subject).to receive(:url_for).with('script_name' => '/cy/alternative_engine').and_return('/cy/alternative_engine')
```
Comment (esbel) [inline spec/controllers/mount_controller_spec.rb]: Would following do the trick?

`expect(subject).to receive(:url_for).with('script_name' => '/cy/alternative_engine').and_call_original`

Comment (asmega) [inline spec/controllers/mount_controller_spec.rb]: unfortunately not. i've not setup all the params so the route fails.

Comment (esbel) [inline spec/controllers/mount_controller_spec.rb]: ok

**Comment by esbel** [comment]
:+1:

**Pull request merged**